src/gptscript.js

```diff
@@ -64,6 +65,15 @@ async function listTools() {
     }
 }
 
+async function version() {
+    try {
+        const version = await run(['--version']);
+        return version;
+    } catch (error) {
+        throw error;
+    }
+}
```

Comment:
This is all just

```
function version() {
  return run(['--version'])
}
```

but with extra steps…

`async function foo() { return await bar() }` is the same as `function foo() { return bar() }`.  run() already returns a promise, just return that.  You're making returning a 2nd promise that does nothing but waits for the first one.

And `try { stuff() } catch (e) { throw e }` is the same as just `stuff()`.  You're catching what was already thrown and then just rethrowing it.  Don't try/catch unless you're 1) going to do something with the error or 2) want the function to continue instead of throwing and terminating.

One or both of these can also be applied to various other functions below.